charts/ocis/values.yaml

```diff
@@ -907,191 +1890,412 @@ services:
     extraLabels: {}
     # Per-service image configuration. Overrides the default setting from `image` if set.
     image:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image repository
-      repository: ""
+      repository:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image tag.
-      tag: ""
+      tag:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image sha / digest (optional).
-      sha: ""
+      sha:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image pull policy
       pullPolicy:
 
   # -- AUTH MACHINE service.
   # @default -- see detailed service configuration options below
   authmachine:
+    # @schema
+    # $ref: schema/templates.json#/resourcesTemplate
+    # @schema
     # -- Per-service resources configuration. Overrides the default setting from `resources` if set.
     resources: {}
+    # @schema
+    # $ref: schema/templates.json#/labelsTemplate
+    # @schema
     # -- Per-service nodeSelector configuration. Overrides the default setting from `nodeSelector` if set.
     nodeSelector: {}
+    # @schema
+    # type: [string, null]
+    # required: false
+    # @schema
     # -- Per-service priorityClassName configuration. Overrides the default setting from `priorityClassName` if set.
-    priorityClassName: ""
+    priorityClassName:
+    # @schema
+    # $ref: schema/templates.json#/podDisruptionBudgetTemplate
+    # @schema
     # -- Per-service PodDisruptionBudget. Overrides the default setting from `podDisruptionBudget` if set.
     podDisruptionBudget: {}
+    # @schema
+    # $ref: schema/templates.json#/autoscalingTemplate
+    # @schema
     # -- Per-service autoscaling. Overrides the default setting from `autoscaling` if set.
     autoscaling: {}
-    # -- Affinity settings for the authmachine service. See the documentation of this setting in approvider for examples.
+    # @schema
+    # $ref: schema/templates.json#/affinityTemplate
+    # @schema
+    # -- Per-service affinity.
     affinity: {}
+    # @schema
+    # $ref: schema/templates.json#/labelsTemplate
+    # @schema
     # -- Per-service custom labels
     extraLabels: {}
     # Per-service image configuration. Overrides the default setting from `image` if set.
     image:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image repository
-      repository: ""
+      repository:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image tag.
-      tag: ""
+      tag:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image sha / digest (optional).
-      sha: ""
+      sha:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
```

Comment:
We should add an enum here with the three values for `pullPolicy`

I stop commenting here. This just should be applied to all services.

Reply:
same as https://github.com/owncloud/ocis-charts/pull/811/files#r1921904014